assets/src/edit-story/components/library/panes/text/textPresets.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { dataFontEm } from '../../../../units';
+import { TEXT_ELEMENT_DEFAULT_FONT } from '../../../../app/font/defaultFonts';
+
+const DEFAULT_PRESET = {
+  id: 'default-text',
+  content: `<span style="font-weight: 400">${__(
+    'Fill in some text',
+    'web-stories'
+  )}</span>`,
+  fontSize: dataFontEm(1),
```

Comment:
Where does this value come from btw?
For example, on my 13'' screen, using `dataFontEm(1)` will display tiny, probably the real size in pixels will be less than 10px. That might not be even accessible anymore due to the small font size depending on the screen.

Assuming the Story is viewed in a mobile device it's likely that this will make the default quite small in the FE as well.

Reply:
FI: Tested it out on my 13'' screen:
- The real font size in the editor is `7.63636px`
- In the FE checking with the Chrome device toolbar for iPhone X, the real font size is around `12px`.

Reply:
1) In regards to using 1 em: 
I found some sources saying that the defualt pixel size is usually 16px (e.g. https://www.w3schools.com/tags/ref_pxtoemconversion.asp). So I assumed because the specs said 16px in the issue, that we were supposed to put in 1em so that it scales according to device. (16px on a screen with high pixel density would be extremely small)

2) In regards to em in general:
Shouldn't EM scale based on how small your screen is? So in a small screen the em should rescale to a value based on your screen size and pixel density?

Regardless the other body text in that file are currently using 1.1 em and it already exists in the current codebase. You can have a look at the current story editor and see what the body font preset looks like on your 13 in screen to test this out. If it actually does look very small, then it might be worth rethinking all of the fonts sizes in general that use em?

Reply:
Also I realised I didn't see your second comment before I commented the above. I think if the real font editor on the current build seems small for you this is an issue we should bring to the team and see how to best approach it.

Reply:
Yep, saw the `1.1em` for the body text and that, too, seemed to be quite tiny in the editor on a smaller screen. Perhaps the idea for the "body text" is to fit a lot of text on a Page which might be fine in the FE then.

For the default Text element, this value definitely seems too small though and not really accessible, it's quite eye-straining on my 13'', and also looks too small when testing FE with mobile.

Perhaps it could make sense to just test which font size would look good for a default Text element on the FE, e.g. for some popular mobile device and set the default value based on that?

Thoughts?

Reply:
`1em` is currently about `5%` of the page's height, or should be `14px` in data pixels. Depending on the actual screen size is could be any value, but it's always about `5%` of the page's height.

Reply:
If 1em should be about 14px in pixels, there might be a larger issue with the em sizes if @miina saw it at 7.6px. It's possible the current DEFAULT_EM calculation is not working very well in smaller screen sizes. I think we should open a new issue about this as it affects all fonts with font size in em.

Reply:
Considering that 1em is 14px in pixels, should I just put in 1.1em for this PR, and we can open a new issue about fixing font sizes in general in smaller screens to investigate this further? I'd prefer the default text font to be relative to the rest of the fonts and then fix the em value, rather than just have this one be really big compared to the other ones while we wait to fix the rest.

I think this is beyond the scope of this if the em value itself is too small. Thoughts?

Reply:
I added the font size of 1.5 em as per conversation on slack with @miina PTAL.

Reply:
@miina In your initial comment, what value are you looking at to get to the 10px value? @o-fernandez responsiveness is an issue that requires spec-ing out and prioritize.

Reply:
@samitron7 In the editor, for getting the `7.63636px` font size, the used value for the default font was `1em` (14 as the font size in Design Panel). In case of larger screens, the font looks larger as well.

Since we're not using actual pixels in the editor then it's not that straightforward to set the default value to `16px`. It might be good to specify the device (or viewport size) where the font size in pixels should be `16` in the front-end -- since on different devices the font size will differ as well. Thoughts?

Reply:
IMHO we should keep this at `1` for now. `1em` for us simply means the "default size". Because the story is proportionally responsive, the meaning of `1em` should not change based on how much the editor is zoomed in/out, or how much the output AMP story is zoomed in/out. Currently, `1em` is set to be about `5%` of the story page, i.e. that maps to ~14 lines of text that would fit on the page. If we like some other value that would work better as a default, let's pick one and update it for the whole editor. We'll just need to change a single [`DEFAULT_EM` constant](https://github.com/google/web-stories-wp/blob/b93f5b8d33e17eec5e76c4cc975951f90a809313/assets/src/edit-story/constants.js#L46). As such, I see this problem mostly separate from the original bug report. And, the value of `DEFAULT_EM` should definitely be decided by @samitron7.

Reply:
I agree with @dvoytenko that it should be 1em for now since then everything is proportional. I can change it back to 1em if @miina is happy with it. LMK what you think @miina

Reply:
I agree that from our side it makes sense to have it `1em`, however, from the product's perspective it does not make sense to have a default value which is not usable on smaller screens. As long as it gets fixed soon, it's OK by me but let's follow up on this with @samitron7 to ensure it's not forgotten.

Reply:
Added the 1em fix. PTAL.